csi/volumeutils.py

```diff
@@ -207,6 +208,14 @@ def create_virtblock_volume(hostvol_mnt, volname, size):
         path=os.path.dirname(volpath)
     ))
 
+    if os.path.exists(volpath_full):
+        rand = uuid.uuid1()
```

Comment:
I think adding `int(time.time())` is good compared to random UUID. It helps to identify the volume based on the time when it was rolled over.

Reply:
Done